packages/sdk-codegen/src/specConverter.ts

```diff
@@ -26,7 +26,8 @@
 
 import { isEmpty } from 'lodash'
 import { APIMethods } from '@looker/sdk-rtl'
-import { ApiModel, ArgValues, IApiModel } from './sdkModels'
+import { IApiVersion, IApiVersionElement } from '@looker/sdk'
```

Comment:
seems a little counter-direction to be importing parts of our looker specific API spec here, no? Is there a generic/standard version/schema for this defined somewhere? If not, I'd suggest we define our own local copy here even if it's an exact replica of the Looker interfaces - that way this package isn't relying on `@looker/sdk`

Reply:
Yeah, I think I actually did that duplication in another source file. Reliance on `@looker/sdk-rtl` is fine, but it shouldn't rely at all on `@looker/sdk`. Then we can document that "versions" payload as something people could use to generate their own specs.